src/main/java/uk/gov/hmcts/reform/sandl/snlevents/service/ActionService.java

```diff
@@ -19,6 +21,9 @@
     @Autowired
     RulesService rulesService;
 
+    @Autowired
+    ActivityLogRepository activityLogRepository;
```

Comment:
"Use the most restrictive access level that makes sense for a particular member. Use private unless you have a good reason not to." https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html also suggested reading on data hiding i.e. https://techdifferences.com/difference-between-abstraction-and-data-hiding.html